<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFixValue</class>
 <widget class="QDialog" name="DialogFixValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>881</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>压力数据修正</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修正压力值超过</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editMax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>kg的通道</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>通道压力修正为原值+修正值</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>全部通道修正</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtComm">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道1修正值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道2修正值</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道3修正值</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道4修正值</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道5修正值</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道6修正值</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道7修正值</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道8修正值</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道9修正值</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道10修正值</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道11修正值</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>通道12修正值</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-520</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnFix">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开始修正</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
